app/models/responses/FlashKeys.scala

```diff
@@ -18,5 +18,6 @@ package models.responses
 
 object FlashKeys {
   val lrn: String = "LRN"
+  val decType: String = "DECTYPE"
```

Comment:
I'll use as value `DecType` the same approach like `expiryDate` not `lrn`

Reply:
Good point.  I should be able to change them all to camelCase.   Will (hopefully) flush out anywhere that is not using these constants (like confirmation page wasn't)

Reply:
2nd thought - initial case.  And "LRN" is probably OK as it's an abbreviation.

Reply:
Yeah, `lrn` is correct